frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -481,6 +481,10 @@ function ReaderBookmark:onShowBookmark(match_table)
     local multilines_show_more_text = G_reader_settings:isTrue("bookmarks_items_multilines_show_more_text")
     local show_separator = G_reader_settings:isTrue("bookmarks_items_show_separator")
 
+    self.bookmark_menu = CenterContainer:new{
+        dimen = Screen:getSize(),
+        covers_fullscreen = true, -- hint for UIManager:_repaint()
+    }
     local bm_menu = Menu:new{
```

Comment:
Couldn't you just make this one self.bookmark_menu?

Reply:
I think it could be just enough - but there's still the covers_fullscreen to plug into menu, and ReaderToc and other to update the same way if we want to stay consistent. And the risk of some other unseen issue. Too late, going the easy road :)